pkgs/by-name/ic/icestorm/package.nix

```
@@ -18,44 +19,42 @@
   usePyPy ? stdenv.hostPlatform.system == "x86_64-linux",
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: rec {
```

if you're making use of finalAttrs, delete rec completely
```suggestion
stdenv.mkDerivation (finalAttrs: {
```